### DISCUSSION TITLE
Cannot apply for GitHub Education because of wrong data in billing?

### DISCUSSION THREAD

**Discussion by 2ktan** [community/community#145308] [GitHub Education]
### Hi everyone,

Hi there!
I'm trying to apply for Github Education to get Student Developer Pack, but no matter how much I try, whatever is checking my ID is telling me I put my name wrong on billing when everything is okay, 1:1 from my ID. What should I do? Thanks in advance.
![image](https://github.com/user-attachments/assets/45f3a386-098b-45e3-ab7c-32d7b5e5aa64)

**Comment by OmgRod** [comment]
You have to put your billing information. You don't have to put in your card details, just your name and address. https://github.com/settings/billing/payment_information

**Reply by 2ktan** [reply]
Bro, I did. That's literally the problem. Thanks tho
